src/dos/drive_local.cpp

```diff
@@ -154,9 +154,15 @@ bool localDrive::FileOpen(DOS_File **file, char *name, uint32_t flags)
 		// If yes, check if the file can be opened with Read-only access:
 		fhandle = fopen_wrap(newname, "rb");
 		if (fhandle) {
+			if (!always_allow_rw_open)
+				fhandle = NULL;
 
 #ifdef DEBUG
-			open_msg = "wanted writes but opened read-only";
+			if (can_rw_open_protected_files) {
```

Comment:
Maybe `can_rw_open_protected_files` was a previous name for `always_allow_rw_open`?

Reply:
Wanted to make internal var name more verbose while keeping config var name concise. Still not sure about these, any alternative name suggestions are welcome.

Reply:
Oh, what I meant to say is that `can_rw_open_protected_files` doesn't exist as a variable in the code - so this `DEBUG`-enabled will fail to compile.

It should be replaced with your current `always_allow_rw_open` bool, which I think is nicely self-documenting! :+1: